frappe/desk/reportview.py

```diff
@@ -144,7 +144,8 @@ def export_query():
 
 	data = [['Sr'] + get_labels(db_query.fields, doctype)]
 	for i, row in enumerate(ret):
-		data.append([i+1] + list(row))
+		translated_row = list(map(_, row))
```

Comment:
this is very expensive, can you add `if frappe.local.lang != "en":` ?

Also can you try adding a test?

Reply:
Sure I'll add the check and test.